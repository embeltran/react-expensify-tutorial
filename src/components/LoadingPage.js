import React from 'react';

const LoadingPage = () => (
    <div className="loader">
        <img src="/images/loader.gif" alt="" />
    </div>
);

export default LoadingPage;