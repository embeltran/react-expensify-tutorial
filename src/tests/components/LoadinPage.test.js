import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from '../../components/LoadingPage';

test('should render Login correctly', () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
  });
  