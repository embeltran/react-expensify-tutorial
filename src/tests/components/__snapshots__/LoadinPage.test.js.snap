// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Login correctly 1`] = `
<div
  className="loader"
>
  <img
    alt=""
    src="/images/loader.gif"
  />
</div>
`;
